scripts/jsonify.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import argparse
+import random
+import json
+import sys
+import apply_bpe
+from sacrebleu import extract_ngrams
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--bpe', default='model/bpe.model')
+parser.add_argument('--first-word', '-f', action='store_true', default=False)
+parser.add_argument('--last-word', '-l', action='store_true', default=False)
+parser.add_argument('--random-word', '-r', type=int, default=0)
+parser.add_argument('--constraints', '-c', nargs='*', type=str, default=[])
+parser.add_argument('--phrase-len', '-p', type=int, default=1)
```

Comment:
help missing for all of these arguments.

Reply:
Thanks. This script is going to be removed. I'm merging in 1.18 and will push that up shortly after testing.